File: DofusApi/Models/ForumPost.cs
using System;

namespace DofusApi.Models
{
    public class ForumPost
    {
        public int ID { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public DateTime CreatedAt { get; set; }
        public int ForumCategoryID { get; set; }
        public ForumCategory ForumCategory { get; set; }
        public int? UserID { get; set; }
        public User User { get; set; }
    }
}

File: DofusSpa/src/app/_models/forum-post.ts
import { ForumCategory } from "./forum-category";
import { User } from "./user";

export interface ForumPost{
    id : number,
    title : string,
    content : string,
    createdAt : Date,
    forumCategoryID : number,
    forumCategory : ForumCategory,
    userID : number,
    user : User
}